torch/_inductor/overrides.py

```diff
@@ -28,7 +28,7 @@ class AutogradMonkeypatch(TorchFunctionMode):
     def __torch_function__(self, func, types, args=(), kwargs=None):
         if not kwargs:
             kwargs = {}
-        if func is replacements:
+        if func in replacements:
```

Comment:
@jansel do you think this codepath should be needed?  or if things were working ok before should we just delete AutogradMonkeyPatch?

Reply:
Ideally no, in practice maybe?

The dropout replacements was happening pretty inconsistently, so this was an (obviously broken) attempt to get them to work in a more robust way.  I think the bigger fix is to allow for pre-autograd replacements in AotAutograd.